src/coreclr/src/jit/emit.h

```diff
@@ -2180,9 +2173,9 @@ class emitter
         dataSection*   dsdList;
         dataSection*   dsdLast;
         UNATIVE_OFFSET dsdOffs;
-        bool           align16;
+        UNATIVE_OFFSET alignment;
```

Comment:
A comment would be great, at least saying that these are in bytes.

Reply:
Fixed.